source/getting_started/aiot_intelligent_edge/coco_for_applications/analytics_client_applications/browser_js_analytics_client_applications/api_reference_browser_js_analytics_client_apps.rst

```diff
@@ -3,9 +3,6 @@
 API Reference Documentation
```

Comment:
The png's above should be in an appropriate sub-folder. Please discuss with Akshay on the sub-folders to use.

Reply:
handled